vcd/datasource_vcd_vm_vgpu_policy.go

```diff
@@ -0,0 +1,87 @@
+package vcd
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func datasourceVcdVmVgpuPolicy() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: datasourceVcdVmVgpuPolicyRead,
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"vgpu_profile": {
+				Type:        schema.TypeList,
+				Computed:    true,
+				Description: "Defines the vGPU profile configuration.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"id": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "The identifier of the vGPU profile.",
+						},
+						"count": {
+							Type:        schema.TypeInt,
+							Computed:    true,
+							Description: "Specifies the number of vGPU profiles. ",
```

Comment:
Spacing nit (not sure about the dot removal in descriptions, as we have them in certain resources/data sources)
```suggestion
							Description: "Specifies the number of vGPU profiles",
```

Reply:
Fixed